packages/app/src/components/pages/Home/ApplicationViewByStages.js

```diff
@@ -17,35 +22,65 @@ const ApplicationViewByStages = () => {
     return statuses.map( (status, index) => <option key={index} selected={status === currentlyViewing} value={status}>{status}</option> )
   }
 
+  const renderSortArrow = (columnName) => {
+    if (!sortConfig) {
+      return
+    }
+    return sortConfig.key === columnName ?
+      <FontAwesomeIcon icon={sortConfig.direction === 'ascending' ? "arrow-down" : "arrow-up"} />
+      : null
+  }
+
   return (
     <table className="new-files-table">
       <thead>
         <tr>
           <td className="add-padding-left new-files-title">
             <FontAwesomeIcon icon="file-image" color="green" />
-            <select onChange={e => setCurrentlyViewing(e.target.value)} className="stage-dropdown">
+            <select onChange={e => setCurrentlyViewing(e.currentTarget.value)} className="stage-dropdown">
               {renderOptionList()}
             </select>
           </td>
         </tr>
       </thead>
       <tbody>
         <tr className="home-new-files-headers">
-          <td className="add-padding-left"><h2><strong>Application Name</strong></h2></td>
-          <td><h2><strong>HP Name</strong></h2></td>
-          <td><h2><strong>Family Member Name</strong></h2></td>
-          <td><h2><strong>Received Date</strong></h2></td>
-          <td><h2><strong>Stage</strong></h2></td>
+          <td className="add-padding-left">
+            <h2 onClick={() => requestSort('nominationName')} style={{cursor: 'pointer'}}>
+              <strong>Application Name {renderSortArrow('nominationName')}</strong>
+            </h2>
+          </td>
+          <td>
+            <h2 onClick={() => requestSort('providerName')} style={{cursor: 'pointer'}}>
+              <strong>HP Name {renderSortArrow('providerName')}</strong>
+            </h2>
+          </td>
+          <td>
+            <h2 onClick={() => requestSort('representativeName')} style={{cursor: 'pointer'}}>
+              <strong>Family Member Name {renderSortArrow('representativeName')}</strong>
+            </h2>
+          </td>
+          <td>
+            <h2 onClick={() => requestSort('dateReceived')} style={{cursor: 'pointer'}}>
+              <strong>Received Date {renderSortArrow('dateReceived')}</strong>
+            </h2>
+          </td>
```

Comment:
These a lot being repeated here in these sortable `td`s here. I think we should create a function that takes the key and label and returns the markup for one of these header elements, name it something like `renderSortableCell(key, label)`, and then call that function like so:
```jsx
<td>{ renderSortableCell('providerName', 'HP Name') }</td>
```

Additionally, instead of having `style={{cursor: 'pointer'}}` be an inline style, let's create a CSS classname called something like `.clickable` and put this somewhere in the CSS:

```css
.clickable {
  cursor: 'pointer';
}